src/meta-cli/src/deploy/actors/task/command.rs

```diff
@@ -100,6 +100,8 @@ impl CommandContext {
             path,
         } = self;
 
+        let optional_envs = [("MCLI_PREFIX", shared_config.prefix.clone())];
+
```

Comment:
It's only containing one element, add all `envs` here or just, use `env`.

Reply:
It's just for readability and future use cases.